src/selectors/app.js

```diff
@@ -131,20 +134,42 @@ export const getTimelineHeight: Selector<null | CssPixels> = createSelector(
           height += ACTIVE_TAB_TIMELINE_RESOURCES_HEADER_HEIGHT;
         }
 
-        for (const [
-          trackIndex,
-          globalTrack,
-        ] of activeTabGlobalTracks.entries()) {
-          if (!hiddenGlobalTracks.has(trackIndex)) {
-            switch (globalTrack.type) {
-              case 'screenshots':
-                height += screenshotTrackHeight + border;
-                break;
-              case 'tab':
+        for (const globalTrack of activeTabGlobalTracks) {
+          switch (globalTrack.type) {
+            case 'screenshots':
+              height += screenshotTrackHeight + border;
+              break;
+            case 'tab':
+              {
+                // The thread tracks have enough complexity that it warrants measuring
```

Comment:
Not for this PR:

I wonder if there is a better way to just measure everything, and remove this function.